File: katalyze/src/main/java/messageHandlers/NullHandler.java
package messageHandlers;

import org.apache.log4j.Logger;

import legacyfeed.SimpleMessage;

public class NullHandler extends SingleMessageHandler {
	static Logger logger = Logger.getLogger(SingleMessageHandler.class);

	String tag; 
	
	public NullHandler(String tag) {
		super(tag);
	}

	@Override
	public void process(SimpleMessage message) {
		logger.debug(String.format("Ignoring message %s", message));
	}

}

File: katalyze/src/main/kotlin/messageHandlers/NullHandler.kt
package messageHandlers

import legacyfeed.SimpleMessage
import org.apache.logging.log4j.kotlin.logger

class NullHandler(tag: String) : SingleMessageHandler(tag) {
    override fun process(message: SimpleMessage) {
        logger.debug(String.format("Ignoring message %s", message))
    }

    companion object {
        private val logger = logger()
    }
}